src/classes/UTIL_IntegrationConfig.cls

```diff
@@ -0,0 +1,147 @@
+                callableApi = initCallableApi();
+            }
+            return callableApi;
+        }
+        set;
+    }
+
+    /**********************************************************************************************************
+    * @description Checks if the package is installed
+    */
+    protected abstract Boolean initIsInstalled();
+
+    /**********************************************************************************************************
+    * @description Determines the package namespace
+    */
+    protected abstract String initNamespace();
+
+    /**********************************************************************************************************
+    * @description Confirms if the (un)managed package is installed
+    */
+    public Boolean isInstalled() {
+        return isInstalled;
+    }
+
+    /**********************************************************************************************************
+    * @description Instantiate the Callable API endpoint in the package
+    */
+    public System.Callable getCallableApi() {
+        return callableApi;
+    }
+
+    /**********************************************************************************************************
+    * @description Instantiate the Callable API endpoint in the package
+    */
+    private System.Callable initCallableApi() {
+        if (!isInstalled) {
+            return null;
+        }
+
+        return (Callable) Type.forName(namespace, callableApiClassName).newInstance();
+    }
+
+    /**********************************************************************************************************
+    * @description Adds the namespace before an SObject name or a child relationship name
+    */
+    public String prefixNamespace(String entity) {
+        if (String.isBlank(namespace)) {
+            return entity;
+        }
+
+        return namespace + '__' + entity;
+    }
+
+    public class DsoNpspPackage extends UTIL_IntegrationConfig {
+        /**********************************************************************************************************
+        * @description Constructor
+        */
+        public DsoNpspPackage() {
+            callableApiClassName = 'BridgeCallable_API';
+        }
+
+        /**********************************************************************************************************
+        * @description Checks if the package is installed
+        */
+        protected override Boolean initIsInstalled() {
+            return System.Type.forName(prefixNamespace('ArchiveRollupSummary__b')) != null;
+        }
+
+        /**********************************************************************************************************
+        * @description Determines the package namespace
+        */
+        protected override String initNamespace() {
+            if (System.Type.forName(DEFAULT_NPSP_DSO_BRIDGE_NAMESPACE + '__ArchiveRollupSummary__b') != null) {
```

Comment:
Can the constant be pulled into the inner class (without `static` or used here as `final String` local var)?